compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -452,6 +501,34 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
     }
 
+    fn suggested_tuple_wrap(
+        &self,
+        expected_input_tys: &[Ty<'tcx>],
+        provided_args: &'tcx [hir::Expr<'tcx>],
+    ) -> Option<FnArgsAsTuple<'_>> {
+        let [expected_arg_type] = &expected_input_tys[..] else { return None };
+
+        let ty::Tuple(expected_elems) = self.resolve_vars_if_possible(*expected_arg_type).kind() else { return None };
```

Comment:
small nit: Could you add a line break before the `else`? This line is really long.

Reply:
Sure, done :)

Out of interest, I notice running `x.py`'s `tidy` doesn't change this either way - is it just that let-else isn't currently handled by it?

Reply:
Yeah, IIRC rustfmt doesn't format `let_else` yet. It's a very new feature :)